sky/utils/schemas.py

```diff
@@ -1269,6 +1272,22 @@ def get_config_schema():
                     },
                     'additionalProperties': False,
                 },
+                'nebius': {
+                    'type': 'object',
+                    'required': [],
+                    'properties': {
+                        'credentials_file_path': {
```

Comment:
We need to update the doc for the remote API server deployment with Nebius credentials?